container-dependency-versions/pom.xml

```diff
@@ -383,6 +383,11 @@
                 <artifactId>xml-apis</artifactId>
                 <version>1.4.01</version>
             </dependency>
+            <dependency>
```

Comment:
This must be moved to parent/pom.xml. This pom is only for dependencies that are exposed by the container to bundles.

Reply:
Moved in https://github.com/vespa-engine/vespa/pull/8879 along with the compile scope change.